assets/css/table.css

```diff
@@ -32,6 +32,10 @@
   background-repeat: no-repeat;
 }
 
+.instructor_dashboard_table_link {
+  @apply h-14 text-gray-500 font-semibold text-xs sm:text-sm;
+}
+
```

Comment:
NIT: I do not know why the styling of the table got corrupted and we now have two arrow icons shown in the sorted column, since we did not change any of the past css that were applied. Anyway, if we want to fix that we just need to change line 13 to be:
`.instructor_dashboard_table th i {`